introduction/writing-style.md

```diff
@@ -0,0 +1,13 @@
+# Writing style
```

Comment:
@dontpanicgr could we maybe make the title reflect the capitilization issue? Like "Writing style: Capitilization"

Reply:
Good suggestion. We'll fix it.